builder/bundleset.go

```diff
@@ -31,7 +31,7 @@ type bundle struct {
 	Files map[string]bool
 
 	/* hidden property, not to be included in file usr/share/clear/allbundles */
-	AllRpmPackages map[string]bool `json:"-"`
+	AllRpmPackages map[string][]string `json:"-"`
```

Comment:
Also, instead of `AllRpmPackages map[string][]string`, see if you can define it as `AllRpmPackages map[string]packageMetadata`. Then you can access the values as `pkg.name` and `pkg.repo` instead of `pkg[0]` and `pkg[2]`.

Maybe even consider renaming `AllRpmPackages` to `AllRpms` :)

Reply:
`var fileSystemRpm []string` uses the same string array structure. It would be good to use the `packageMetadata` struct there too.